### PR TITLE
Add Srgb render target support

### PR DESCRIPTION
**Pull request by bartwe**

### CODE REVIEW

**Review by flibitijibibo — CHANGES_REQUESTED** [review]
Need to undo the submodule update and fix tabbing, otherwise fine

**Comment by bartwe** [comment]
How do you mean 'undo the submodule?' it contains required changes

**Comment by flibitijibibo** [comment]
We have to use the revision that gets merges into the master branch first - I'll merge that first, then we can use that revision for this patch.

File: src/Graphics/SurfaceFormat.cs [hunk 1]
```diff
@@ -98,5 +98,10 @@ public enum SurfaceFormat
 		/// Unsigned 32-bit ABGR pixel format for store 8 bits per channel (XNA3)
 		/// </summary>
 		ColorBgraEXT,
-	}
+		/// <summary>
+		/// Unsigned 32-bit ARGB pixel format for store 8 bits per channel.
+		/// Byte encoding is in sRGB colorspace, read in shader in linear colorspace.
+		/// </summary>
+		ColorSrgbEXT,
+    }
```
Comment (flibitijibibo) [inline src/Graphics/SurfaceFormat.cs:106]:
```suggestion
	}
```

File: src/Graphics/GraphicsAdapter.cs [hunk 2]
```diff
@@ -206,6 +206,7 @@ public bool IsProfileSupported(GraphicsProfile graphicsProfile)
 			 * -flibit
 			 */
 			if (	format != SurfaceFormat.Color &&
+				format != SurfaceFormat.ColorSrgbEXT &&
```
Comment (flibitijibibo) [inline src/Graphics/GraphicsAdapter.cs:209]: Should be at the bottom of the list

**Review by flibitijibibo — CHANGES_REQUESTED** [review]
Couple minor things...

**Comment by flibitijibibo** [comment]
Wrote in the bare minimum since this took too long... should be enough unless someone's hacking into ProfileCapabilities:

https://github.com/FNA-XNA/FNA/commit/7fb834b7ad59d81d2b3eb783f7673fb1d0c6e3e5

**Pull request closed without merging**